### DISCUSSION TITLE
Ts tgl terrastation 1 pro

### DISCUSSION THREAD

**Discussion by HWIIC1987** [1000001101000/Debian_on_Buffalo#210] [General]
Hi has anyone manged to get debian on device

**Comment by 1000001101000** [comment]
In 2020 I managed to boot into Debian on mine but ran into issues with the sata controller/driver. The combination of that and how much a pain it was to even boot lead me to move on to other things.

**Reply by HWIIC1987** [reply]
Thank you was just hoping to bring life back to my device I have tsxl and a l220d both running debian and then installed omv over the top and both support 10tb plus drives thank you
